scripts/switcher.rollup.config.js

```diff
@@ -17,6 +20,7 @@ export default {
     format: 'umd'
   },
   plugins: [
+    json(),
```

Comment:
I think this was previously hidden, since rollup was going into a screen, `package.json` from `videojs-http-streaming` was not being imported correctly.